src/plotter.py

```diff
@@ -155,6 +155,68 @@ def make_table_plot_from_dict(self, this_dict,
         plt.close()
 
 
+    def make_sentiment_plot(self, this_dict,
+                            tosave=False):
+
+        # Get the data and labels
+        question = this_dict['question']
+        values   = this_dict['responses']['values']
+        keys     = this_dict['responses']['keys']
+        labels   = this_dict['labels']
+        n_rows   = len(keys)
+
+        # Create subplots
+        fig, axes = plt.subplots(nrows=n_rows, figsize=(10, 3*n_rows))
+        fig.suptitle(question, fontsize=16, x=0.063, ha='left')
+
+        # Create histogram for each row
+        for i in range(n_rows):
+
+            # Make the histogram
+            # Create bars with custom style
+            axes[i].hist(values[:,i], bins=np.arange(0.5, 6.5, 1),
+                        color=VF_BLUE, alpha=1.0,
+                        edgecolor=VF_BLUE, linewidth=1.5,
+                        rwidth=0.8)
+
+            # Add value labels on top of each bar
+            for rect in axes[i].patches:
+                height = rect.get_height()
+                axes[i].text(rect.get_x() + rect.get_width()/2, height,
+                             f'{int(height)}',
+                             ha='center', va='bottom')
+
+            # Annotations
+            mean = np.nanmean(values[:,i])
+            std  = np.nanstd(values[:,i])
+
+            axes[i].annotate(f'n = {len(values[:,i])}\nMean: {mean:.2f}\nStd: {std:.2f}',
+                            xy=(0.02, 0.73),
+                            color='gray',
+                            xycoords='axes fraction',
+                            bbox=dict(
+                                facecolor='white',
+                                edgecolor='none',
+                                alpha=0.7)
+                            )
+
+            axes[i].set_title(f'"{keys[i]}"', loc='left',
+                              color=VF_BLUE_DARK, pad=10)
+
+            axes[i].set_ylabel('Count')
+            axes[i].set_xticks(range(1,6))
+            axes[i].set_ylim(top=axes[i].get_ylim()[1] * 1.1)
+            axes[i].set_xticklabels(labels)
+
+        plt.tight_layout()
+
+        if tosave:
```

Comment:
`tosave` flag seems to be called `saveas` in the other functions

Reply:
Thanks for noticing. It was sort of intentional. `tosomething` is a boolean. `saveas` is actually a string.